<ui version="4.0" >
 <class>graphInspector</class>
 <widget class="QWidget" name="graphInspector" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="leftMargin" >
    <number>2</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>2</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>1</number>
     </property>
     <item>
      <widget class="QComboBox" name="graphBox" />
     </item>
     <item>
      <widget class="QPushButton" name="addEdit" >
       <property name="minimumSize" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text" >
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeEdit" >
       <property name="minimumSize" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text" >
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" >
     <property name="horizontalSpacing" >
      <number>1</number>
     </property>
     <property name="verticalSpacing" >
      <number>1</number>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="lbl2" >
       <property name="text" >
        <string>Название</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2" >
      <widget class="QLineEdit" name="descriptionEdit" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Толщина графика</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2" >
      <widget class="QSlider" name="widthEdit" >
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="maximum" >
        <number>5</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Цвет графика</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="2" >
      <widget class="QPushButton" name="colorEdit" >
       <property name="autoFillBackground" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0" >
      <widget class="QCheckBox" name="drawableEdit" >
       <property name="text" >
        <string>Видимый</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="styleLabel" >
       <property name="text" >
        <string>Стиль</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2" >
      <widget class="QComboBox" name="styleEdit" />
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>207</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
